Formula/e/energy.rb

```diff
@@ -0,0 +1,25 @@
+class Energy < Formula
+  desc "CLI is used to initialize the Energy development environment tools"
+  homepage "https://energye.github.io"
+  url "https://github.com/energye/energy/archive/refs/tags/v2.4.5.tar.gz"
```

Comment:
The version of tarball is 2.4.5 and version of cli which can be obtained from the static file of homepage is 1.0.10. I think then the version of formula will be 2.4.5 that is not match with cli binary.

Reply:
The mismatch between the CLI version number and Formula has been resolved